openupgrade_scripts/scripts/hr_attendance/16.0.2.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Models in module 'hr_attendance'---
+---Fields in module 'hr_attendance'---
+hr_attendance / res.company              / attendance_barcode_source (selection): NEW selection_keys: ['back', 'front', 'scanner'], hasdefault: default
+hr_attendance / res.company              / attendance_kiosk_delay (integer): NEW hasdefault: default
```

Comment:
The new default is different from the default behavior in v15.

Here the stored value is multiplied by 1000 in v16:

https://github.com/odoo/odoo/blob/17151f5c369dce7ca5d38c5fd0db8dceab5cfab7/addons/hr_attendance/models/hr_employee.py#L165

and here it's used:

https://github.com/odoo/odoo/blob/17151f5c369dce7ca5d38c5fd0db8dceab5cfab7/addons/hr_attendance/static/src/js/greeting_message.js#L86-L87

and in v15, the default value was 5000:

https://github.com/odoo/odoo/blob/17151f5c369dce7ca5d38c5fd0db8dceab5cfab7/addons/hr_attendance/static/src/js/greeting_message.js#L86-L87

So, to preserve previous behavior, we should set in all existing companies the value 5. Anyway I think they have changed this value for giving more time to read the message, so it's an acceptable change of behavior, and more being able to be configured later, but the important thing is to document all of this here.